libqtile/window.py

```diff
@@ -955,7 +956,7 @@ def toggleminimize(self):
         warnings.warn("toggleminimize is deprecated, use toggle_minimize", DeprecationWarning)
         self.toggle_minimize()
 
-    def cmd_static(self, screen, x=None, y=None, width=None, height=None):
+    def cmd_static(self, screen=None, x=None, y=None, width=None, height=None):
```

Comment:
Is this part of the patch necessary? It seems like it is not.

Reply:
This is not necessary, but without this it will be required to specify screen.

Reply:
Right, but it wasn't before, so is there a reason to relax this requirement?

Reply:
Well, in my view if someone will want to use it(place static window not on a certain screen, but on current screen) he have to write extra lines of code, on other if someone want to use specific screen he already specify it.

Reply:
Yep, fair point. So this looks fine, can you add a test for the whole thing?

Reply:
Um, I don't think I can handle it :disappointed_relieved: